nav2_smac_planner/src/smac_planner_lattice.cpp

```diff
@@ -160,6 +176,13 @@ void SmacPlannerLattice::configure(
     _max_iterations = std::numeric_limits<int>::max();
   }
 
+  if (_coarse_search_resolution <= 0) {
+    RCLCPP_WARN(
+      _logger, "coarse iteration resolution selected as <= 0, "
+      "disabling coarse iteration resolution search for goal heading"
+    );
+    _coarse_search_resolution = 0;
```

Comment:
Here and in the other file (and in the dynamic parameters): Shouldn't this be 1? I don't think `0` is a valid configuration. I know you error check for `0` to replace with `1`, but I think we should still set `1`

Reply:
done in https://github.com/ros-navigation/navigation2/pull/4127/commits/009a8a9df2186fa18c2f67fa047982b002a0780c